### PR TITLE
feat(expressive): update button component to expressive theme

### PR DESCRIPTION
**Pull request by annawen1**
### Related Ticket(s)

Expressive theme – Icon-only Buttons #823
Expressive theme – Button #821

### Description

Edit the button padding to fit the dimensions expected for expressive theme.

Details in the related tickets.

### Changelog

**Changed**

- changed the storybook to import 20x20 icons instead of 16x16
- adjusted the padding for the buttons

### CODE REVIEW

File: packages/styles/scss/themes/expressive/components/_button.scss [hunk 1]
```diff
@@ -15,7 +15,43 @@
...
+    }
+  }
+
+  .#{$prefix}--btn--field,
   .#{$prefix}--btn--sm {
-    min-height: rem(40px);
+    padding: 6px 61px 6px 13px;
+
+    &.#{$prefix}--btn--ghost {
+      padding: 6px 13px;
```
Comment (jeffchew) [inline packages/styles/scss/themes/expressive/components/_button.scss]: @annawen1 @oliviaflory this is some pretty specific padding, will we run into issues when we try to contribute this back to carbon?

Comment (annawen1) [reply packages/styles/scss/themes/expressive/components/_button.scss]: hmm I could convert it to rems? 

Although I do see specific padding on their end too:
```
/// @type Value
/// @access public
/// @group button
$button-padding-field: calc(0.675rem - 3px) 60px calc(0.675rem - 3px) 12px !default;
```

Comment (jeffchew) [reply packages/styles/scss/themes/expressive/components/_button.scss]: Looks like they set a default $button-padding-field variable that's used on their end. Would we be able to set variables on our end (to match whatever variables they are using) and set it on our end?

Comment (annawen1) [reply packages/styles/scss/themes/expressive/components/_button.scss]: yeah I could do that

Comment (annawen1) [reply packages/styles/scss/themes/expressive/components/_button.scss]: yeah i could do that

**Pull request merged**